app/views/tips/edit.html.erb

```diff
@@ -1,13 +1,17 @@
-<h1 align= "center">Update your tip!</h1>
+<h1 align= "center" class="tip-header">Update your tip!</h1>
 
-<center>
-  <%= form_tag "/pieces/tips/#{@tip.id}", method: :patch do |f| %>
-    <%= label_tag :tip, "Edit your practice tip for this piece" %>
-    <%= text_field_tag :tip, "#{@tip.tip}" %>
-    <br>
-    <%= label_tag :difficulty_rating, "Difficulty Rating - 1: Pretty Easy, 5: Highly difficult" %>
-    <%= select_tag :difficulty_rating, options_for_select([1, 2, 3, 4, 5]) %>
-    <br>
-    <%= submit_tag "Update Tip" %>
+<section class="form create-tip">
+  <%= form_tag "/pieces/tips/#{@tip.id}", method: :patch do %>
+    <div class="form-group">
+      <%= label_tag :tip %>
+      <%= text_area_tag :tip, @tip.tip, class: 'form-control', rows: 8 %>
+    </div>
+
+    <div class="form-group">
```

Comment:
Does this still use the label tag showing users what 1 and 5 mean?

Reply:
We can do that in the next sprint!